src/components/networks/utils.js

```diff
@@ -29,6 +29,46 @@ export function validateIpv4(address) {
     return ip.isV4Format(address);
 }
 
+/**
+ * Valides if the user provided broadcast address
+ * Using broacast address of network space as the address of virtual network's bridge
+ * is forbidden and would disable the connectivity of the virtual network.
+ *
+ * @param {string} address
+ * @returns {boolean}
+ */
+export function ipv4IsBroadcast(address, netMask) {
+    if (!validateNetmask(netMask))
+        return false;
+    const mask = netmaskConvert(netMask);
+    const subnet = ip.subnet(address, mask);
+
+    console.log(address);
+    console.log(subnet.broadcastAddress);
+    // user provided netmask
+    return address === subnet.broadcastAddress;
+}
+
+/**
+ * Valides if the user provided network identifier as the address
+ * Using network identifier is however forbidden
+ * During virtual network creation, address is assigned to bridge
+ * But no interface can have the address same as the network identifier,
```

Comment:
So I would rewrite the function docstring to not be so verbose. Also I think this is a helper/util, we don't care about what Is allowed or not in the UI here, we just want to check one thing, if an address is the Network Identifier. 

So:
```
Returns if the provided address is the network's identifier
```  

If you want to mention about the limitations, these comments belong to the place where you use this function.

Reply:
Fixed this and all above